### ISSUE TITLE
SVG Timeline

### ISSUE THREAD

**Issue by alnopa9** [dorothealint/William_Combe_Works#17]
@ebeshero I am having a difficult time getting my SVG timeline to be the right size. The way the site renders it now, more than 3/4 of the timeline gets cut off. I thought it was a viewBox setting, but nothing I change in the CSS or SVG itself seems to allow it to be seen. Any help would be appreciated.
Thanks!
TImeline Html file:
https://github.com/dorothealint/William_Combe_Works/blob/master/Combe%20Webpage/timeline.html
Combe CSS file:
https://github.com/dorothealint/William_Combe_Works/blob/master/Combe%20Webpage/combe.css
Timeline SVG file:
https://github.com/dorothealint/William_Combe_Works/blob/master/Combe%20Webpage/timelineSVG.svg?short_path=2f042c9

**Comment by ebeshero** [comment]
@alnopa9 Yikes. I tried a few simple things to try to change your viewport, but I'm tangled up in the complexity of the CSS and HTML surrounding the SVG, so I don't think I can give you an answer quickly--sorry. But what I can understand more readily is the SVG file on its own--and the viewport there is configured in a way I'm not familiar with. I think you *do* want to work with a viewport here, and embedding SVG in HTML is a little complicated, but based on the dimensions of the "boxes" on the DOM that you're working with. I think you *want* your timeline to be scrollable, but the space I see on the HTML page is certainly smaller than you need. 

Okay, so to work through this, I recommend we both consult Sarah Soueidan's (very awesome!) tutorial on Understanding SVG Coordinate Systems and Transformations:
[Part 1: viewport, viewBox, and preserveAspectRatio](https://www.sarasoueidan.com/blog/svg-coordinate-systems/)
[Part 2: The transform attribute](https://www.sarasoueidan.com/blog/svg-transformations/)
[Part 3: Establishing new viewports](https://www.sarasoueidan.com/blog/nesting-svgs/)

I also recommend (sorry!) simplifying your CSS. I'm not sure how the different levels of div are interacting with one another, and I suspect the complexity here is coming from lots of experimentation, but some of those experiments might be conflicting with one another. It will help me if you can explain something of what you want to be doing, and point out things in the HTML structure and CSS rules that are controlling things you definitely want... I wonder if we can find a way to simplify this.

**Comment by alnopa9** [comment]
Okay. I have been meaning to clean up the CSS because I know there are a lot of duplicate settings. I'm starting on that now and then I will take a look at the links you shared.

**Comment by alnopa9** [comment]
@ebeshero @dorothealint I got the timeline up on the site and it's fully visible now. Once I cleaned up the CSS and read the Part 1 tutorial, I set the height and width of the viewport using pixels as the measurement instead of percent and that fixed it. There is a drop down menu on the left to quickly navigate through the years and the works of his that we have on the site are a darker blue. I used the Overholt reference in the feedback issue to create links to their pages as well.
